card/lib/card/format.rb

```diff
@@ -292,6 +293,13 @@ def rescue_view e, view
       end
     end
 
+    def current_view view
```

Comment:
Unused method argument - `view`. If it's necessary, use `_` or `_view` as an argument name to indicate that it won't be used. You can also write as `current_view(*)` if you want the method to accept any arguments but don't care about them.